LICENSE

```diff
@@ -1,5 +1,3 @@
-ISC License 
```

Comment:
This will cause license autodetection to fail in GitHub. It needs to be added back.

Reply:
Are you just referring to this when you go to the license or is there something else to it?
![image](https://user-images.githubusercontent.com/455257/135365129-db7e620d-521a-4f8b-859d-ac050cf3b848.png)
It wasn't visible because I removed the commit from my develop branch.

Reply:
Actually, do we want this or #3930?

I'm fine with either, but if we go with this, the comment I left needs to be addressed.